app/app/src/main/java/org/omecproject/up4/cli/FlowsStatsCommand.java

```diff
@@ -0,0 +1,108 @@
+        adminService.getDownlinkFlows().forEach(dlUpfFlow -> {
+            beforeDownlink.put(dlUpfFlow.getSession().ueAddress(), dlUpfFlow.getCounter());
+        });
+
+        Map<Ip4Address, UpfCounter> beforeUplink = Maps.newHashMap();
+        adminService.getUplinkFlows().forEach(ulUpfFlow -> {
+            beforeUplink.put(ulUpfFlow.getTermination().ueSessionId(), ulUpfFlow.getCounter());
+        });
+        Thread.sleep(SLEEP_TIME_MS);
+
+        Map<Ip4Address, UpfCounter> afterDownlink = Maps.newHashMap();
+        adminService.getDownlinkFlows().forEach(dlUpfFlow -> {
+            afterDownlink.put(dlUpfFlow.getSession().ueAddress(), dlUpfFlow.getCounter());
+        });
+
+        Map<Ip4Address, UpfCounter> afterUplink = Maps.newHashMap();
+        adminService.getUplinkFlows().forEach(ulUpfFlow -> {
+            afterUplink.put(ulUpfFlow.getTermination().ueSessionId(), ulUpfFlow.getCounter());
+        });
+
+        print(SEPARATOR);
+        if (!(beforeDownlink.keySet().equals(beforeUplink.keySet()) &&
+                beforeDownlink.keySet().equals(afterUplink) &&
+                beforeDownlink.keySet().equals(afterDownlink))) {
+            print("ERROR while reading stats!");
+            return;
+        }
+        for (Ip4Address ueSessionId : beforeDownlink.keySet()) {
+            if (!beforeUplink.containsKey(ueSessionId) ||
+                    !afterDownlink.containsKey(ueSessionId) ||
+                    !afterUplink.containsKey(ueSessionId)) {
+                print("UE session: " + ueSessionId.toString());
+                UpfCounter bfUl = beforeUplink.get(ueSessionId);
+                UpfCounter bfDl = beforeDownlink.get(ueSessionId);
+                UpfCounter afUl = afterUplink.get(ueSessionId);
+                UpfCounter afDl = afterDownlink.get(ueSessionId);
+                long droppedPktsUl = (afUl.getIngressPkts() - afUl.getEgressPkts()) -
+                        (bfUl.getIngressPkts() - bfUl.getEgressPkts());
+                long droppedPktsDl = (afDl.getIngressPkts() - afDl.getEgressPkts()) -
+                        (bfDl.getIngressPkts() - bfDl.getEgressPkts());
+
+                long txPktsUl = (afUl.getEgressPkts() - bfUl.getEgressPkts()) / SLEEP_TIME_MS;
+                long txBytesUl = (afUl.getEgressBytes() - bfUl.getEgressBytes()) / SLEEP_TIME_MS;
+                long txPktsDl = (afDl.getEgressPkts() - bfDl.getEgressPkts()) / SLEEP_TIME_MS;
+                long txBytesDl = (afDl.getEgressBytes() - bfDl.getEgressBytes()) / SLEEP_TIME_MS;
+
+                long rxPktsUl = (afUl.getIngressPkts() - bfUl.getIngressPkts()) / SLEEP_TIME_MS;
+                long rxBytesUl = (afUl.getIngressBytes() - bfUl.getIngressBytes()) / SLEEP_TIME_MS;
+                long rxPktsDl = (afDl.getIngressPkts() - bfDl.getIngressPkts()) / SLEEP_TIME_MS;
+                long rxBytesDl = (afDl.getIngressBytes() - bfDl.getIngressBytes()) / SLEEP_TIME_MS;
+
+                print("UPLINK");
```

Comment:
All caps is generally interpreted as shouting, and there's nothing important about traffic direction. In general I recommend weavoid using all caps unless absolutely necessary (e.g., for acronym, or important error messages). So it's ok to print just "Uplink" and "Downlink".
```suggestion
                print("Uplink");
```